llvm/test/CodeGen/AArch64/sme-lazy-save-call.ll

```diff
@@ -46,13 +47,14 @@ define void @test_lazy_save_2_callees() nounwind "aarch64_inout_za" {
 ; CHECK-NEXT:    mov x29, sp
 ; CHECK-NEXT:    sub sp, sp, #16
 ; CHECK-NEXT:    rdsvl x19, #1
-; CHECK-NEXT:    mov x8, sp
-; CHECK-NEXT:    msub x8, x19, x19, x8
+; CHECK-NEXT:    mov x9, sp
+; CHECK-NEXT:    mul x8, x19, x19
+; CHECK-NEXT:    sub x8, x9, x8
```

Comment:
This is an unfortunate regression. If you pass `SVL` to the ExpandZA pseudo, rather than SVL * SVL, you can do the multiply and subtract using MSUB.